### ISSUE TITLE
Debug DLLs being merely present crashes non Steam versions of ZUN games available on Steam

### ISSUE THREAD

**Issue by 32th-System** [thpatch/thcrap#116]
[Video demonstration](https://youtu.be/1aQUgLVzJSQ)

According to the Visual Studio debugger this crash happens in line 97 of steam.cpp in a debug build

[thcrap_log.txt](https://github.com/thpatch/thcrap/files/3478517/thcrap_log.txt)

**Comment by brliron** [comment]
This is a known bug, but I'm not sure we have an open issue on Github.

Statically loaded DLLs don't do anything, but when it comes to plugins, stars fall from the sky, forests start burning, and kittens start getting killed. What happens is that both versions of the plugin get loaded (they both end with .dll, and they both have a function called `thcrap_init`). So, everything happens twice. And some things are *not* supposed to happen twice.
If it didn't crash on the steamAPI_Init call, it would have crashed later.

**Comment by DankRank** [comment]
tl;dr don't mix debug and release versions

**Comment by brliron** [comment]
Fixed for the general case with 584391f55a816ae640f19d27fcd2d0e3b4b8290a. Please reopen if there are issues related to the Steam versions (or, of course, if the fix didn't work).
